### DISCUSSION TITLE
Need help to extract the visible text in a Web Page.

### DISCUSSION THREAD

**Discussion by Vinoth997** [webdriverio/webdriverio#11719] [Community]
I'm new to WebDriverIO.

I'm now dealing with an issue in which a locator contains the CSS property.

**text-overflow: ellipsis;**

This allows the text in the locator to fit inside the width.

As an example:
"Long live the king" is written on the locator.

The text will be displayed as "Long live t..." after using CSS text-flow.

My concern is, utilising webdriverIO typescript, is there any way to ensure that the option does not contain three dots? Using the CSS text-overflow property

or is there another way to deal with this?

FYI: This work is about a dropdown and a list on text init, and I need to test that the whole text is displayed or multiple dots are displayed.

**Comment by christian-bromann** [comment]
> "Long live the king" is written on the locator.

What locator?

**Reply by Vinoth997** [reply]
"'div class="dropdown-optiontext">Long live the king</div'"

im using css selector '.dropdown-optiontext'

**Reply by christian-bromann** [reply]
You are using a CSS selector which means the tendered text inside the element is not important.
